_includes/user-menu.html

```diff
@@ -1,101 +1,130 @@
+      <li><a href="{{docu}}/configuration/ui-based.html">Paper UI Configuration</a></li>
+      <li><a href="{{docu}}/configuration/rules-ng.html">Next-Gen Rules</a></li>
+      <li><a href="{{docu}}/configuration/ui-based.html">HABmin</a></li>
+      <li><a href="{{docu}}/configuration/ui-based.html">HABPanel</a></li>
+      <hr />
+      <li><a href="{{docu}}/configuration/multimedia.html">Audio &amp; Voice</a></li>
     </ul>
   </li>
   <li><a href="{{docu}}/addons/index.html">Add-ons</a>
     <ul>
       <li><a href="{{docu}}/addons/index.html">Overview</a></li>
+      <hr />
       <li><a href="{{docu}}/addons/bindings.html">Bindings</a>
         <ul>
+          <li><a href="{{docu}}/addons/bindings.html">Overview</a></li>
+          <hr />
           {% for binding in site.data.bindings %}
           <li><a href="{{docu}}/addons/bindings/{{ binding.id }}/readme.html">{{ binding.label }}</a></li>
           {% endfor %}
         </ul>
       </li>
+      <li><a href="{{docu}}/addons/uis.html">User Interfaces</a>
+        <ul>
+          <li><a href="{{docu}}/addons/uis.html">Overview</a></li>
+          <hr />
+          <li><a href="{{docu}}/addons/uis/paper/readme.html">Paper UI</a></li>
+          <li><a href="{{docu}}/addons/uis/habmin/readme.html">HABmin</a></li>
+          <hr />
+          <li><a href="{{docu}}/addons/uis/basic/readme.html">Basic UI</a></li>
+          <li><a href="{{docu}}/addons/uis/habpanel/readme.html">HABPanel</a></li>
+          <li><a href="{{docu}}/addons/uis/classic/readme.html">Classic UI</a></li>
+          <hr />
+          <li><a href="{{docu}}/addons/iconsets/classic/readme.html">Iconset (classic)</a></li>
+        </ul>
+      </li>
+      <li><a href="{{docu}}/addons/persistence.html">Persistence</a></li>
+      <li><a href="{{docu}}/addons/actions.html">Actions</a></li>
+      <li><a href="{{docu}}/addons/transformations.html">Transformations</a></li>
       <li><a href="{{docu}}/addons/voice.html">Voice Services</a>
         <ul>
+          <li><a href="{{docu}}/addons/voice.html">Overview</a></li>
+          <hr />
           {% for voice in site.data.voice %}
           <li><a href="{{docu}}/addons/voice/{{ voice.id }}/readme.html">{{ voice.label }}</a></li>
           {% endfor %}
         </ul>
       </li>
-      <li><a href="{{docu}}/addons/io.html">System Integrations</a>
+      <hr />
+      <li><a href="{{docu}}/addons/io.html">3rd Party Integration</a>
         <ul>
+          <li><a href="{{docu}}/addons/io.html">Overview</a></li>
+          <hr />
           <li><a href="{{docu}}/addons/io/homekit/readme.html">HomeKit</a></li>
-          <li><a href="{{docu}}/addons/io/hueemulation/readme.html">Amazon Echo</a></li>
+          <li><a href="{{docu}}/addons/io/hueemulation/readme.html">Hue Emulation</a></li>
           <li><a href="{{docu}}/addons/io/imperihome/readme.html">ImperiHome</a></li>
+          <li><a href="https://my.openhab.org/docs">my.openHAB (external)</a></li>
```

Comment:
change to `{{docu}}/addons/io/openhabcloud/readme.html` and label to "openHAB Cloud"